src/swish/script-testing.ss

```diff
@@ -63,6 +64,27 @@
   (define swish-exe
     (get-real-path (path-combine (prereq-path) (fix-exe "swish"))))
 
+  (define (ensure-dlls)
+    (define (ctime fn)
+      (match (get-stat fn)
+        [`(<stat> [ctime (,sec . ,nsec)]) (make-time 'time-duration nsec sec)]
```

Comment:
Would you prefer to use mtime instead of ctime? It may not matter in this case.

Reply:
Switched to `mtime`.